platforms/scripts/afu_platform_config

```diff
@@ -785,16 +786,15 @@ def findPlatforms(db_path):
     # Walk all the directories
     for db_dir in db_path:
         # Look for JSON files in each directory
-        for fn in os.listdir(db_dir):
-            if fn.endswith('.json'):
-                try:
-                    with open(os.path.join(db_dir, fn)) as f:
-                        # Does it have a platform name field?
-                        db = json.load(f)
-                        platforms.add(db['platform-name'])
-                except Exception:
-                    # Give up on this file or directory if there is any error
-                    None
+        for json_file in glob.glob(os.path.join(db_dir, "*.json")):
+            try:
+                with open(json_file, 'r') as f:
+                    # Does it have a platform name field?
+                    db = json.load(f)
+                    platforms.add(db['platform-name'])
+            except Exception:
+                # Give up on this file if there is any error
+                None
```

Comment:
Maybe this could be refactored to one common function?

```python
def find_json_db(db_path):
    for db_dir in db_path:
        for json_file in glob.glob(os.path.join(db_dir, "*.json")):
            try:
                with open(json_file, 'r') as fd:
                    db = json.load(fd)
            except Exception:
                pass
            finally:
                yield db
```

Reply:
I did think about it, but it would get ugly. They are different classes of JSON files and the validation tests are different.

Reply:
Just saying, in case you want to handle exceptions more explicitly, you would only do that in one spot rather than different ones.
Handling explicit exceptions might be a good idea too...
 `IOError` vs. `KeyError`/`ValueError`

Reply:
You're welcome to make changes if you think they are important and for the better. I'm going to move on to getting MCP builds working on DCP, which was the goal of this.